reviews/review-01+2020-11-02.md

```diff
@@ -0,0 +1,305 @@
+    there isn't a strong reason to split (although the separation
+    between 2.8 and 2.9 is one possible place to do so); I would mainly
+    recommend splitting this chapter so that there isn't so much content
+    before getting to exercises.
+
+-   The section on package documentation (13.6) seems like it could be
+    its own chapter, as it delves fairly deeply into the nuts and bolts
+    of package documentation tools, which is a departure from the rest
+    of the chapter's focus on packaging as a whole. I can understand
+    wanting to include the documentation section in the packaging
+    chapter so it's not an afterthought, but it makes for a chapter that
+    covers two quite distinct topics (conceptually) -- packaging, and
+    good documentation practices. Another option besides splitting the
+    packaging chapter (ch 13) into two would be to put some of the more
+    detailed information about documentation generation into an
+    appendix.
+
+## Titles/subtitles
+
+Are the chapter titles/subtitles appropriate? If not, how can they be
+modified?
+
+I would recommend standardizing the chapter titles to all follow the
+same "formula" of either a) tool/topic, b) intended outcome, or c) a
+combination of the two.
+
+For example, chapter five could be:
+
+a)  Git at the Command Line (current title)
+
+b)  Tracking Changes
+
+c)  Tracking Changes with Git at the Command Line
+
+For chapter eight:
+
+a)  Make
+
+b)  Automating Analyses (current title)
+
+c)  Automating Analyses with Make
+
+It may not be possible to fully standardize the titles (I can't think of
+a good short punchline for the Unix shell, for example), but I think a
+review of the chapter titles to make them align more with the b) and c)
+options would be helpful in clarifying the purpose and big picture topic
+of each chapter.
+
+## Examples and exercises
+
+Are the chapter examples and exercises appropriate to the subject
+matter? If not, how can they be modified?
+
+Yes. One of the book's strengths is the presence of an intriguing and
+accessible sample problem that helps motivate the chapter topics and
+provides a concrete motivation for going through the chapters and
+watching the project grow.
+
+On the whole, the chapter-specific exercises follow directly from the
+chapter content and provide good practice opportunities. In certain
+cases, an exercise requires executing a complex set of steps (ex: 3.8.3,
+4.11.2, or 9.8.6) and could be more valuable if there were hints and/or
+scaffolding to help less experienced readers get started.
+
+## Figures and tables
+
+Are the figures and tables visually appealing and useful for the
+reader?
+
+The screenshots in chapter 6 (Advanced Git) are great, and extremely
+helpful in illustrating what the reader might see in using Github.
+
+All of the figures come at appropriate points in the chapter where a
+figure can aid understanding. I found Figures 2.5 and 2.6 hard to
+understand.
```

Comment:
This comment I think could be dealt with by using a more descriptive caption for the figures. Right now they are very bare.

Reply:
Totally agree that we probably need to take a look at all our figure captions and make sure they are sufficiently descriptive.